test/test_mps.py

```diff
@@ -4397,6 +4411,667 @@ def maybe_transpose(cond, m):
         m2 = maybe_transpose(t3, torch.randn(50, 25, device=device).to(dtype))
         self._test_addmm_addmv(torch.addmm, M, m1, m2, transpose_out=t4)
 
+class TestViewOpsMPS(TestCase):
```

Comment:
Can you please add a big comment here that this bloc should be removed once the corresponding device generic test can be made to run on MPS. It would be good to link exactly which part needs to be updated.
And if anything here was added beyond the original test, I think we want to break it into another class.

Reply:
These are all subset of test/test_view_ops.py tests.